src/Compilers/Core/Portable/InternalUtilities/TextKeyedCache.cs

```diff
@@ -51,13 +51,17 @@ public SharedEntryValue(string Text, T item)
         // local cache
         // simple fast and not threadsafe cache 
         // with limited size and "last add wins" expiration policy
+#pragma warning disable IDE0055 // Fix formatting
         private readonly (string Text, int HashCode, T Item)[] _localTable = new (string Text, int HashCode, T Item)[LocalSize];
+#pragma warning restore IDE0055 // Fix formatting
```

Comment:
What's up with formatting here?

Reply:
Command line builds run with the 15.7 formatter. The 15.8 formatter includes a bug fix for this syntax (space after `new`). There is no way to make both the analyzer and the IDE happy, so the 15.8 style is preferred and the command line warning is suppressed.